lua/tardis/libraries/sh_messages.lua

```diff
@@ -0,0 +1,28 @@
+CreateConVar("tardis2_debug", 0, {FCVAR_ARCHIVE, FCVAR_REPLICATED}, "TARDIS - debug enabled")
+
+function TARDIS:IsDebugOn()
+	return GetConVar("tardis2_debug"):GetBool()
```

Comment:
This should be cached as TARDIS:Debug code may be running multiple times per frame and we do not want to call this every single time as it is relatively expensive, it should be retrieved once to set TARDIS.debug = true/false and updated at runtime using https://wiki.facepunch.com/gmod/cvars.AddChangeCallback